react/Footer/Footer.js

```diff
@@ -59,72 +58,62 @@ export default class Footer extends Component {
-                        { services.map(this.renderLink) }
-                      </ToggleContainer> :
-                      null
-                  }
-                </FooterNav>
-
-                <FooterNav label="Connect">
-                  { connect.map(this.renderLink) }
-                  <ToggleContainer name="SocialToggle" label="Social">
-                    { social.map(this.renderLink) }
-                  </ToggleContainer>
-                </FooterNav>
-
-                <FooterNav secondary label="Employers">
-                  { employers.map(this.renderLink) }
-                </FooterNav>
-              </div>
-
-              <nav className={styles.copyright}>
-                <ScreenReaderOnly>
-                  <h1>Additional Links</h1>
-                </ScreenReaderOnly>
-
+          <div className={styles.content}>
+            <div className={styles.columns}>
+              <FooterNav label="Tools">
+                { tools.map(this.renderLink) }
+                <ToggleContainer name="PartnerSitesToggle" label="SEEK sites">
+                  { seekSites.map(this.renderLink) }
+                </ToggleContainer>
+              </FooterNav>
+
+              <FooterNav secondary label="Company">
+                { company.map(this.renderLink) }
+                <ToggleContainer secondary name="InternationalPartnersToggle" label="International partners">
+                  { partners.map(this.renderLink) }
+                </ToggleContainer>
                 {
-                  copyright.map(({ name, analytics, href, rel, secondary }, key) => (
-                    linkRenderer({
-                      children: name,
-                      href,
-                      rel,
-                      'data-analytics': analytics,
-                      key,
-                      className: classnames({
-                        [styles.copyrightLink]: true,
-                        [styles.secondaryLink]: secondary
-                      })
-                    })
-                  ))
+                  isAU ?
+                    <ToggleContainer secondary name="PartnerServicesToggle" label="Partner services" data-automation="partner-services-toggle">
+                      { services.map(this.renderLink) }
+                    </ToggleContainer> :
+                    null
                 }
-                <p className={styles.copyrightMessage}>{'\u00A9 SEEK. All rights reserved.'}</p>
-              </nav>
+              </FooterNav>
+
+              <FooterNav label="Connect">
+                { connect.map(this.renderLink) }
+                <ToggleContainer name="SocialToggle" label="Social">
+                  { social.map(this.renderLink) }
+                </ToggleContainer>
+              </FooterNav>
+
+              <FooterNav secondary label="Employers">
+                { employers.map(this.renderLink) }
+              </FooterNav>
+            </div>
+
+            <div className={styles.copyright}>
```

Comment:
Can we confirm that this was correctly changed from a `<nav>` to a `<div>`?

Reply:
Yeap, have confirmed with Adem this does not need to be a `<nav>`.